tests/openvino/native/quantization/test_fq_params_calculation.py

```diff
@@ -78,16 +78,16 @@ def fixture_inplace_statistics(request):
 @pytest.mark.parametrize('preset', [QuantizationPreset.PERFORMANCE, QuantizationPreset.MIXED],
                          ids=[QuantizationPreset.PERFORMANCE.value, QuantizationPreset.MIXED.value])
 @pytest.mark.parametrize('model_creator_func', SYNTHETIC_MODELS.values())
-def test_syntetic_models_fq_scales(model_creator_func, preset, inplace_statistics):
+def test_synthetic_models_fq_scales(model_creator_func, preset, inplace_statistics):
     model = model_creator_func()
     quantized_model = quantize_model(model.ov_model, {'preset': preset, 'inplace_statistics': inplace_statistics})
     nodes = get_fq_nodes_stats_algo(quantized_model)
 
     ref_stats_name = model.ref_graph_name.split(".")[0] + f'_{preset.value}.json'
     ref_stats_path = REFERENCE_SCALES_DIR / ref_stats_name
 
-    # Unkomment lines below to generate reference for new models.
-    # from tests.shared.helpers import dump_to_json
+    # Uncomment lines below to generate reference for new models.
+    # from tests.openvino.native.common import dump_to_json
```

Comment:
`tests.openvino.native.common import dump_to_json` or `from tests.shared.helpers import dump_to_json`?

Reply:
Fixed, want to fix only `unkomment`